jetty-runner/src/main/java/org/eclipse/jetty/runner/Runner.java

```diff
@@ -164,6 +170,7 @@ public void usage(String error)
         System.err.println(" --out file                          - info/warn/debug log filename (with optional 'yyyy_mm_dd' wildcard");
         System.err.println(" --host name|ip                      - interface to listen on (default is all interfaces)");
         System.err.println(" --port n                            - port to listen on (default 8080)");
+        System.err.println(" --port-file path                    - file to write the connector port ");
```

Comment:
This description is a bit vague.

Reply:
not sure. what about `file to write a single line with the port used to server http requests` ?

Reply:
@joakime description updated